README.md

```diff
@@ -0,0 +1,63 @@
+### `oz-merkle-tree`
+
+**A Python library to generate Merkle trees and Merkle proofs.**
+
+This is project is compatible with the [OpenZeppelin StandardMerkleTree](https://github.com/OpenZeppelin/merkle-tree).
+
+#### Usage
```

Comment:
would be great to have `Installation` routine section here

Reply:
Added